src/test/java/pro/belbix/ethparser/codegen/SimpleContractGeneratorTest.java

```diff
@@ -90,4 +88,17 @@ void getContract_ZeroEx() {
         () -> assertEquals(5, contract.getFunctions().size(), "Functions size")
     );
   }
+
```

Comment:
please add a test for contract with empty source code
example https://etherscan.io/address/0x0000000000007f150bd6f54c40a34d7c3d5e9f56#code

Reply:
ack